docs/getting-started/install-resoto/docker.md

```diff
@@ -41,7 +41,7 @@ $ mkdir -p resoto/dockerV2
 $ cd resoto
 $ curl -o docker-compose.yaml https://raw.githubusercontent.com/someengineering/resoto/{{latestRelease}}/docker-compose.yaml
 $ curl -o dockerV2/prometheus.yml https://raw.githubusercontent.com/someengineering/resoto/{{latestRelease}}/dockerV2/prometheus.yml
-$ docker compose up -d
+$ docker compose up
```

Comment:
I disagree with the removal of `-d` here. It is not common that users will want to run `docker compose up` not in daemon mode. What is the motivation behind this change?

Reply:
Motivation: people do not see when they can use Resoto and start Resotoshell.

Reply:
@aquamatthias This is not the correct solution to that problem, however. We should instead provide instructions on how to verify that they can start the shell, perhaps by viewing logs?

Reply:
@TheCatLady Can you share your source of information about what is common and what not?

Reply:
@aquamatthias In what case will users want to `docker compose up` _not_ in daemon mode? It is hard to imagine that a user would want this running in the foreground, which requires keeping the terminal open...

Popular publishers of Docker images like LinuxServer.io also instruct users to execute `docker compose up -d`: https://docs.linuxserver.io/general/docker-compose

Reply:
```
docker compose logs -f <service_name>
```
would allow users to view live logs without the possibility of terminating the service if they accidentally close the terminal window.

Reply:
my thinking: people that try it out the first time (getting started) benefit from seeing all the logs. If you put it into prod, you know how to do. On the other hand is a grep more user-friendly. Changed it to grep.

Reply:
`grep` is definitely a more user-friendly option! And I don't know if we should make assumptions about users wanting run in daemon mode or not :)